packages/app/components/claim/claim-button.tsx

```diff
@@ -83,7 +83,7 @@ export const ClaimButton = ({
   const raffleConcludedAt = useMemo(() => {
     if (!isSelf || !isRaffleDrop) return null;
     if (
-      edition.gating_type === "music_presave" &&
+      edition.gating_type === "spotify_presave" &&
```

Comment:
if existing drops are `music_presave` it'll hide the button right? Do you think we need `edition.gating_type === "music_presave" || edition.gating_type === "spotify_presave"`?

Reply:
that's makes sense! we need to compatible with it!